pgtoolkit/hba.py

```diff
@@ -236,6 +236,9 @@ def __str__(self) -> str:
 
         return line
 
+    def __eq__(self, other: object) -> bool:
+        return str(self) == str(other)
+
```

Comment:
Should be in a dedicated commit, with some test. Or did you commit this by mistake?

Reply:
I'll add tests and put it in a separated commit.